typescript/infra/config/environments/mainnet3/warp/configGetters/getEthereumLineaTurtle.ts

```diff
@@ -0,0 +1,39 @@
+import { ethers } from 'ethers';
+
+import { ChainMap, HypTokenRouterConfig, TokenType } from '@hyperlane-xyz/sdk';
+
+import {
+  RouterConfigWithoutOwner,
+  tokens,
+} from '../../../../../src/config/warp.js';
+
+const turtleOwners = {
+  ethereum: '0x2e0355922EF3a5b77d29287C808aEafB4e7f25B2',
+  linea: '0x2e0355922EF3a5b77d29287C808aEafB4e7f25B2',
+};
+
+const ISM_CONFIG = ethers.constants.AddressZero; // Default ISM
```

Comment:
I know that we have this as an example, but we no longer need to set this as address 0. In other words, this can be undefined